.azure-pipelines/mgmt.yml

```diff
@@ -0,0 +1,24 @@
+pr:
+  branches:
+    include:
+    - dev
+
+pool:
+  vmImage: 'VS2017-Win2016'
+
+steps:
+  - task: CmdLine@1
+    inputs:
+      filename: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe
```

Comment:
@shahabhijeet is this still needed? I thought you fully signed the tool dependencies.

Reply:
I'd still like to know whether not this is still necessary. If it isn't we should remove it.

Reply:
@shahabhijeet

Reply:
as of now we need this. When we nugetize our tooling, this will no longer be required.